cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/includes/top.jsp

```diff
@@ -39,6 +39,7 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
The magic. For more information see the two references below.

* [What is Viewport?](https://developer.mozilla.org/en-US/docs/Glossary/viewport)
* [How to use Viewport](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag)